src/lib/expansion/expansion-panel-header.ts

```diff
@@ -101,6 +102,17 @@ export class MatExpansionPanelHeader implements OnDestroy {
     .subscribe(() => this._changeDetectorRef.markForCheck());
 
     _focusMonitor.monitor(_element.nativeElement, renderer, false);
+
+    // When a panel is hosted in an accordion, subscribe to the expand/collapse subject
+    if (this.panel.accordion) {
+      this._expandCollapseAllSubscription =
+        this.panel.accordion.expandCollapseAllSubject.subscribe((expanded: boolean) => {
```

Comment:
The type for `expanded` doesn't need to be given here because it is inferred from the observable.